package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@snowplow/webview-tracker",
+  "version": "0.1.0",
+  "description": "WebView tracker for Snowplow",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/snowplow-incubator/snowplow-webview-tracker"
+  },
+  "contributors": [
+    "Matus Tomlein"
+  ],
+  "main": "dist/index.js",
```

Comment:
This seems to be primarily designed for node applications. The CJS version which is built by default is basically useless for browsers. Given this is for browsers only, and not node, I'd suggest _just_ building the esm version for npm installations, and then having a umd version for everyone else, then also publishing an iife for "tag" usage (and providing a tag).

Reply:
Changed this to follow the approach that I saw used in microbundle:

```json
{
	"main": "./dist/foo.umd.js", // legacy UMD output (for Node & CDN use)
	"module": "./dist/foo.module.mjs", // legacy ES Modules output (for bundlers)
	"exports": "./dist/foo.modern.mjs", // modern ES2017 output
}
```